jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java

```diff
@@ -0,0 +1,34 @@
+package com.google.cloud.tools.jib.maven;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.hamcrest.CoreMatchers;
+import org.junit.Assert;
+
+public class SkippedGoalVerifier {
+
+  public static void verifyGoalIsSkipped(TestProject testProject, String goal)
+      throws VerificationException, IOException {
+    String targetImage = "neverbuilt:maven" + System.nanoTime();
+
+    Verifier verifier = new Verifier(testProject.getProjectRoot().toString());
+    verifier.setSystemProperty("_TARGET_IMAGE", targetImage);
```

Comment:
I think we can just remove this line?

Reply:
without target image specified, it will fail?